Helper/DeviceView.php

```diff
@@ -66,11 +63,17 @@ class DeviceView
     protected $switchParam = self::SWITCH_PARAM_DEFAULT;
 
     /**
+     * @var array
+     */
+    protected $configRedirect;
+
+    /**
      * Constructor
      *
-     * @param \Symfony\Component\DependencyInjection\Container $serviceContainer
+     * @param RequestStack $requestStack
+     * @param array        $configRedirect
      */
-    public function __construct(RequestStack $requestStack = null)
+    public function __construct(RequestStack $requestStack = null, $configRedirect = null)
```

Comment:
@orions and may be you move injection of `$configRedirect` from contructor  to call method? Similar as https://github.com/orions/MobileDetectBundle/blob/fd04f0c36395b86e0985f9c69f97d2be9ea1ccd5/Resources/config/mobile_detect.xml#L22
Something as:
```
<call method="setRedirectConfig">
    <argument>%mobile_detect. redirect%</argument>
</call>
```
That there was a similar config injection logic...